shopelectro/tests/tests_views.py

```diff
@@ -527,20 +531,33 @@ def test_tags_pagination_has_canonical_links(self):
             )
         )
 
-    def test_category_matrix_page(self):
```

Comment:
It looks like an accidentally removed test

Reply:
@artemiy312 , it's just moved to `shopelectro.tests.tests_views.CategoriesMatrix#test_roots_sorting` with another task. I just removed code old doubling